topic-operator/src/main/java/io/strimzi/operator/topic/KafkaStreamsTopicStoreService.java

```diff
@@ -110,8 +111,18 @@ private TopicStore createKafkaTopicStore(Config config, Properties kafkaProperti
             }
         };
 
-        Topology topology = new TopicStoreTopologyProvider(storeTopic, storeName, kafkaProperties, dispatcher).get();
-        streams = new KafkaStreams(topology, kafkaProperties);
+        // provide some Kafka Streams defaults
+        Properties streamsProperties = new Properties();
+        streamsProperties.putAll(kafkaProperties); // don't add them as defaults?
+        Object rf = kafkaProperties.get(StreamsConfig.REPLICATION_FACTOR_CONFIG);
```

Comment:
Who and where configures the `kafkaProperties` that we don't know what it contains?

Reply:
It's configured in Session, out of Config which is setup in Main.

Reply:
Thanks 👍